metadata-db-app/src/cmr/metadata_db/data/oracle/generic_documents.clj

```diff
@@ -101,37 +126,50 @@
         next (+ 1200000001 raw-count)]
     (format "X%s-%s" next provider-id)))
 
-;; WORKS 
+(defn get-concept-id
+  "Convert a native-id to a concept-id"
+  [db concept-type provider native-id]
+  ;; sql will return a map of the column name and value, strip the first value out
+  (-> db
+      (jdbc/query [(str "SELECT concept_id "
+                        "FROM cmr_generic_documents "
+                        "WHERE provider_id = ? AND native_id = ?")
+                   (:provider-id provider) native-id])
+      first
+      vals
+      first))
+
 (defn save-concept
   "Create the document in the database, try to pull as many values out of the
    document as can be found. All documents must have at least a :Name and a
    :MetadataSpecification field."
   [db provider-id document]
-  (let [rm-fields [:concept-id :concept-type :created-at :native-id :provider-id :revision-date :revision-id]
+  (let [rm-fields [:deleted :concept-id :concept-type :created-at :native-id :provider-id :revision-date :revision-id]
         metadata (json/generate-string (strip-fields document rm-fields))
-        metadata-spec (get document "MetadataSpecification")
-        schema (get metadata-spec "Name")
-        version (get metadata-spec "Version")
-        now (coerce/to-sql-time (dtp/parse-datetime (str (tkeep/now))))
+        metadata-spec (get document :MetadataSpecification)
+        schema (get metadata-spec :Name)
+        version (get metadata-spec :Version)
         created-at (coerce/to-sql-time (dtp/parse-datetime (:created-at document)))
         revision-date (coerce/to-sql-time (dtp/parse-datetime (:revision-date document)))]
     (insert-record db
-                   {:id (get-next-id-seq db)
+                   (cutil/tee {:id (get-next-id-seq db)
```

Comment:
Replace with debug logging instead of the println

Reply:
yes, I don't need to commit this